torch/_functorch/aot_autograd.py

```diff
@@ -3003,20 +3004,37 @@ def call_compiled_backward():
 
                     placeholder_list = fx_placeholder_vals(bw_module)
 
+                    forward_saved_for_backwards_strides = None
+                    if fwd_output_strides is not None:
+                        forward_saved_for_backwards_strides = fwd_output_strides[
+                            CompiledFunction.metadata.tensors_saved_for_backwards_slice
+                        ]
+
                     # saved activations can have different stride to eager if
                     # the compiler does layout optimization. We should restride the
                     # tensor passed in for compiling the backward graph using the
                     # saved tensor's stride.
                     for i in range(len(placeholder_list)):
                         ph_arg = placeholder_list[i]
-                        real_arg = all_args[i]
                         if not isinstance(ph_arg, torch.Tensor):
                             continue
 
-                        # Comparing ph_arg.stride() with real_arg.stride() directly may
+                        if forward_saved_for_backwards_strides is None:
```

Comment:
Move this check out of the for loop may be slightly faster.

Reply:
I was trying to save myself an indent 😂  This is only during compilation so I don't think it matters much